spec/models/user_spec.rb

```diff
@@ -63,6 +63,14 @@
     end
   end
 
+  describe "#age_end_of_tax_year" do
+    let(:user) { build :user, birth_date: "1990-04-21" }
+
+    it "returns their age at the end of 2019" do
```

Comment:
Since birth_date can be nil on intakes before we get ID.me back, can you also add a test for that case?